mod-source-record-manager-server/src/main/java/org/folio/rest/impl/ChangeManagerImpl.java

```diff
@@ -204,7 +206,14 @@ public void postChangeManagerJobExecutionsRecordsById(String id, boolean acceptI
         eventDrivenChunkProcessingService.processChunk(entity, id, acceptInstanceId, params)
           .map(processed -> PostChangeManagerJobExecutionsRecordsByIdResponse.respond204())
           .map(Response.class::cast)
-          .otherwise(ExceptionHelper::mapExceptionToResponse)
+          .otherwise(ex -> {
+            if (ex instanceof DuplicateEventException) {
+              LOGGER.warn("postChangeManagerJobExecutionsRecordsById:: Failed to process chunk of RawRecords with JobExecution id {}: {}", id, ex.getMessage());
```

Comment:
Please add jobExecutionId and recordId to the logs where possible.

Reply:
@Aliaksandr-Fedasiuk Thank you! Added "entity.getId()" (RawRecordsId) to this message.